taskcluster/scripts/pipeline/train-taskcluster.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
praise: I think it was a great idea to handle all of the training continuation bits in this script instead of trying to have `train.sh` support it in addition to its core job.

idea: Is there a use case for supporting training continuation outside of Taskcluster? If so, we may want to get an issue on file for making this script Taskcluster agnostic. (I explicitly don't want to add that work to this PR even if it may be useful.)

Reply:
Snakemake already supports this because it works with files. This work is Taskcluster specific, so this looks like a good place to handle it.